protos/Makefile

```diff
@@ -26,16 +30,31 @@ help:
 .PHONY: clean
 clean:
 	@mkdir -p pb && rm -f pb/pb.pb.go
+	@rm -f $(GOBIN)/protoc-gen-go
```

Comment:
`regenerate` is doing `check clean golang_protobuf gogo_protobuf` and `clean` is Phony. So every step will run every time which may be fine because we won't be running make regenerate too often. Could be problematic while I making changes to the proto file though.